app/models/medium.rb

```diff
@@ -49,12 +49,14 @@ class Medium < ApplicationRecord
 
   has_many :quiz_certificates, foreign_key: 'quiz_id', dependent: :destroy
 
-  has_one :assignment
+  has_many :assignments
```

Comment:
Why is this changed and not documented?

Reply:
I guess I forgot - the `has_one` version is actually a bug. Currently you can create multiple assignments for a single medium without being stopped - it is simply not checked. And I think there is no reason why one medium should not be able to have more than one assignment.